src/quantum/README.rst

```diff
@@ -0,0 +1,161 @@
+Microsoft Azure CLI 'quantum' Extension
+==========================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit visit:
+https://docs.microsoft.com/quantum/
+
+
+# Using the `az quantum` extension to list and manage jobs in Azure Quantum
+
+1. Log in to Azure using your credentials.
+   ```dotnetcli
+   az login
+   ```
+>[!NOTE] In case you have more than one subscription associated with your Azure account you must specify the 
+>subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+2. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
+   ```dotnetcli
+   az quantum workspace list
```

Comment:
I get `[]`. Maybe we can start with how to create something. Then we list or show them.